ws/README.md

```diff
@@ -9,18 +9,16 @@ WebSockets server for JSON-RPC 2.0.
 
 ```
 [dependencies]
-jsonrpc-core = "6.0"
-jsonrpc-ws-server = "6.0"
+jsonrpc-ws-server = { git = "https://github.com/paritytech/jsonrpc" }
```

Comment:
Is it okay to fetch git dependency here?

Reply:
yup!